app/src/main/java/com/worldtreeinc/leaves/activity/LoginActivity.java

```diff
@@ -45,7 +46,10 @@ public void onClick(View v) {
                 case R.id.FacebookLoginButton:
                     userAuthentication.FacebookLogin();
                     break;
-            }
+                case R.id.resetPassword:
+                    Intent resetPassword = new Intent(this, ResetPasswordActivity.class);
+                    startActivity(resetPassword);
+                 }
```

Comment:
NITPICK: Alignment issues, the closing brace to end the switch statement should match with the beginning of the switch statement.